pom.xml

```diff
@@ -60,21 +60,21 @@
         <top.level.dir>${project.basedir}/..</top.level.dir>
         <compiler.source>1.8</compiler.source>
         <compiler.target>1.8</compiler.target>
-        <log4j2.version>2.17.0</log4j2.version>
+        <log4j2.version>2.18.0</log4j2.version>
         <commons.configuration.version>1.10</commons.configuration.version>
-        <commons.configuration2.version>2.3</commons.configuration2.version>
+        <commons.configuration2.version>2.8.0</commons.configuration2.version>
         <commons.beanutils.version>1.9.4</commons.beanutils.version>
         <commons.collections.version>3.2.2</commons.collections.version>
         <commons.io.version>2.7</commons.io.version>
-        <commons.codec.version>1.11</commons.codec.version>
-        <guava.version>25.1-jre</guava.version>
+        <commons.codec.version>1.13</commons.codec.version>
+        <guava.version>30.0-jre</guava.version>
         <javax.json.version>1.0</javax.json.version>
         <jsr305.version>3.0.1</jsr305.version>
         <javassist.version>3.28.0-GA</javassist.version>
         <jersey.version>3.0.3</jersey.version>
         <jersey.hk2.version>3.0.3</jersey.hk2.version>
-        <jackson.version>2.12.1</jackson.version>
-        <junit.version>4.12</junit.version>
+        <jackson.version>2.14.0-rc1</jackson.version>
+        <junit.version>4.13.1</junit.version>
```

Comment:
contains a lot of CVE problems alerts (high impact)

and ignore `commons.collections.version` (3.x) now because it changed package name in version `4.x` (someone could address it in future)